Mathlib/Algebra/Homology/ShortComplex/ModuleCat.lean

```diff
@@ -117,61 +117,78 @@ def moduleCatLeftHomologyData : S.LeftHomologyData where
   wπ := by aesop
   hπ := ModuleCat.cokernelIsColimit (ModuleCat.ofHom S.moduleCatToCycles)
 
+lemma moduleCatLeftHomologyData_K :
+    S.moduleCatLeftHomologyData.K = ModuleCat.of R (LinearMap.ker S.g.hom) :=
+  rfl
```

Comment:
Is it possible to make `simps` make this lemma, and make it not be a simp lemma, whilst still having it make `i_hom` etc as simp lemmas?

Reply:
Ooh, apparently yes, thank you